theseus/utils/examples/pose_graph/dataset.py

```diff
@@ -282,10 +282,10 @@ def generate_synthetic_3D(
             noise_relative_pose = th.SE3.exp_map(
                 torch.cat(
                     [
-                        rotation_noise
-                        * (2 * torch.rand(dataset_size, 3, dtype=dtype) - 1),
                         translation_noise
                         * (2.0 * torch.rand(dataset_size, 3, dtype=dtype) - 1),
+                        rotation_noise
```

Comment:
Does line 274 need updating as well?

```python
                torch.cat(
                    [
                        2.0 * torch.rand(dataset_size, 3, dtype=dtype)
                        - 1,  # translation
                        torch.rand(dataset_size, 3, dtype=dtype) - 0.5,  # rotation
                    ],
```

Reply:
Not necessarily since this is only related to the noises.